### PR TITLE
[For 2.6.x] Allow a custom Tracim repo to build the Docker image

### LINKED ISSUE #2753: The repo URL is hardcoded in the Dockerfile
## Bug description & symptom

When attempting to build the Docker image from a fork of Tracim, the Docker recipe still fetches the main Tracim repository. 

## Expected

It should be possible to build the Docker image from a local repository, or from a fork, without editing the Dockerfile.

## Suggestion

~By default, the docker image could clone the `tracim/.git` local repository instead of re-cloning the repo from internet. We may not want this, though.~ Actually, this is impossible without bind-mounting or copying the repository inside the Docker container.

## Impacted components

- deployment with Docker

### PR DESCRIPTION
**Pull request by raphj**
Related to #2753.

## Checkpoints

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

**Comment by Skylsmoi** [comment]
I added @buxx and @PhilippeAccorsi as reviewers

File: tools_docker/README.md [hunk 1]
```diff
@@ -164,4 +164,10 @@ You can also build image with specific tag (This build is make just with necessa
 
 Ex: `docker build --build-arg TAG="release_02.00.00" -t algoo/tracim:release_02.00.00 .`
 
+By default, the Docker image is built from the main repository of Tracim. To clone Tracim from another repository, use the REPO argument. Don't forget to set a suitable image name.
+
+Ex: `docker build --build-arg REPO=~/repos/tracim/.git -t algoo/tracim:localrepo .`
```
Comment (PhilippeAccorsi) [inline tools_docker/README.md]:
```suggestion
Ex: `docker build --build-arg REPO="~/repos/tracim/.git" -t algoo/tracim:localrepo .`
```
If user have space or something like that, its better to add double quote.

File: tools_docker/README.md [hunk 2]
```diff
@@ -164,4 +164,10 @@ You can also build image with specific tag (This build is make just with necessa
...
+
+
+
 ⚠ **Its not possible to build image with ARG TAG and ARG BRANCH in same time.**
```
Comment (PhilippeAccorsi) [inline tools_docker/README.md]: This line is very important, so I think its better to put this line just before the line starting by `By default, the Docker......`

File: tools_docker/README.md (hunk 1 above)
Comment (inkhey) [reply tools_docker/README.md]: fixe

File: tools_docker/README.md (hunk 2 above)
Comment (inkhey) [reply tools_docker/README.md]: Reorganised in a more clear way.

File: tools_docker/README.md [hunk 3]
```diff
@@ -150,6 +150,10 @@ To build image
     cd tools_docker/Debian_Uwsgi
     docker build -t algoo/tracim:latest .
 
+#### With Custom Branch or Tag
+
+⚠ **Its not possible to build image with ARG TAG and ARG BRANCH in same time.**
```
Comment (raphj) [inline tools_docker/README.md:155]:
```suggestion
⚠ **It is not possible to build an image with both ARG TAG and ARG BRANCH at same time.**
```

File: tools_docker/README.md [hunk 4]
```diff
@@ -164,4 +168,8 @@ You can also build image with specific tag (This build is make just with necessa
 
 Ex: `docker build --build-arg TAG="release_02.00.00" -t algoo/tracim:release_02.00.00 .`
 
-⚠ **Its not possible to build image with ARG TAG and ARG BRANCH in same time.**
+#### With Custom Repository
+
+By default, the Docker image is built from the main repository of Tracim. To clone Tracim from another repository, use the REPO argument. Don't forget to set a suitable image name.
+
+Ex: `docker build --build-arg REPO="~/repos/tracim/.git" -t algoo/tracim:localrepo .`
```
Comment (raphj) [inline tools_docker/README.md:175]:
```suggestion
Ex: `docker build --build-arg REPO="https://github.com/<me>/tracim.git" -t algoo/tracim:myrepo .`
```

**Pull request merged**